lib/rubocop/cop/rspec/hooks_before_examples.rb

```diff
@@ -38,14 +38,12 @@ class HooksBeforeExamples < Base
           }
         PATTERN
 
-        def on_block(node)
+        def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
           return unless example_group_with_body?(node)
 
           check_hooks(node.body) if multiline_block?(node.body)
         end
 
-        alias on_numblock on_block
```

Comment:
This had no effect, so I removed it. It relies on a previous block and then simply uses `next_sibling` together with `any_block` (which includes `itblock`). I only added tests to show it already works.